server/gitrest/packages/gitrest-base/src/utils/isomorphicgitManager.ts

```diff
@@ -375,11 +376,12 @@ export class IsomorphicGitManagerFactory extends RepositoryManagerFactoryBase<vo
     }
 
     protected async initGitRepo(fs: IFileSystemManager, gitdir: string): Promise<void> {
-        return isomorphicGit.init({
-            fs,
-            gitdir,
-            bare: true,
-        });
+        // return isomorphicGit.init({
+        //     fs,
+        //     gitdir,
+        //     bare: true,
+        // });
```

Comment:
neat implementation of `slimInit`! Do you think we could then remove these commented-out lines?

Reply:
Good catch! Thanks!

Reply:
I ended up keeping the old init function alternative and hiding the new one behind a flag. This way we can easily revert if it is problematic for reasons that I didn't reproduce locally.